dbt_subprojects/dex/macros/models/_project/zeroex/zeroex_settler_addresses.sql

```diff
@@ -20,6 +20,11 @@ tbl_addresses AS (
         contract_address = 0x00000000000004533Fe15556B1E086BB1A72cEae
         -- Filter for the specific event signature (ProtocolFeeCollectorAddress event)
         AND topic0 = 0xaa94c583a45742b26ac5274d230aea34ab334ed5722264aa5673010e612bc0b2
+        {% if is_incremental() %}
+            AND {{ incremental_predicate('block_time') }}
+        {% else %}
+            AND block_time >= DATE '2024-03-10'
+        {% endif %}
```

Comment:
nice, wasn't sure since there are a few `begin_` column names, but also saw `tx_hash` so figured normal incremental may work. this will help prod run times 🙏